libcxx/test/libcxx/utilities/utility/pairs/pairs.pair/trivially_copyable.compile.pass.cpp

```diff
@@ -0,0 +1,95 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: gcc && (c++11 || c++14 || c++17)
```

Comment:
We should add tests for the GCC half of this, rather than leaving it untested.

Reply:
Or at minimum there should be a comment describing _why_ GCC isn't supported here. Because it's not at all obvious.

Reply:
I forgot to remove that condition from an earlier version. Thanks for noticing.